core/utils/arp.h

```diff
@@ -0,0 +1,42 @@
+#ifndef BESS_UTILS_ARP_H_
+#define BESS_UTILS_ARP_H_
+
+#include "endian.h"
+#include "ether.h"
+#include "ip.h"
+
+namespace bess {
+namespace utils {
+
+// A basic ARP header definition.
+struct[[gnu::packed]] ARP {
+  // Ethernet hardware format for hrd
+  static const be16_t kEtherHardwareFormat = 1;
+
+  enum Opcode : uint16_t {
+    REQUEST = 1,
+    REPLY = 2,
+    REVREQUEST = 3,
+    REVREPLY = 4,
+    INVREQUEST = 8,
+    INVREPLY = 8,
+  };
+
+  be16_t hrd;   // format of hardware address
```

Comment:
Can we either:
* modify field names to be more descriptive, or
* add a comment that these field names are from RFC documents, so that others can get less tempted to rename the fields later...